dashboard/app/controllers/ops/workshop_attendance_controller.rb

```diff
@@ -53,6 +53,62 @@ def workshop
       end
     end
 
+    # GET /ops/cohorts/1/teachers
+    def attendance
+      @workshop = Workshop.includes(segments: :attendances).find(params.require(:workshop_id))
+
+      respond_with (@workshop.teachers) do |format|
+        format.csv do
+          #Specify filename
+          response.headers['Content-Disposition'] = 'attachment; filename="' + @workshop.name + '-Attendance.csv"'
+
+          # Generate csv column headers dynamically
+          header = ["User ID", "First Name", "Last Name", "E-mail", "District Name", "School Name"]
+          segment_number = 0
+          @workshop.segments.each { |segment|
+            header << (segment.start.to_date.to_s + ", " + segment.start.strftime("%H:%M").to_s + " - " + segment.end.strftime("%H:%M").to_s)
+            header << ("Segment " + (segment_number + 1).to_s + " notes")
```

Comment:
if you use this style of string: `"Segment #{segment_number + 1} notes"` you don't have to to_s everything